### DISCUSSION TITLE
ECHIPA BINARII

### DISCUSSION THREAD

**Discussion by sebiboga** [peviitor-ro/community#103] [TEAMs]
Aici este echipa BINARII

**Comment by sebiboga** [comment]
Rares Irimus
role: scraper developer
discord: RaresCode

**Reply by RaresCode** [reply]
1x Automation Tester(SDET) NEEDED
1 x Scraper developer with Java (NEEDED)

**Reply by RaresCode** [reply]
Team Manager

**Comment by RaresCode** [comment]
George Luncasu
role: Team Lead and Automation Tester
discord: georgeluncasu

**Comment by RaresCode** [comment]
Cosmina Ciobanu
role: Manual Tester
discord: ciobanucosmina_48165

**Comment by RaresCode** [comment]
Larisa Pălimaru
role: Automation Tester
discord: larisa8361

**Comment by RaresCode** [comment]
Doina Pipas
role: Manual Tester
discord: doinapipas09

**Comment by RaresCode** [comment]
Sarra Creţa
role: Manual Tester
discord: sarra_20287

**Comment by AdrianaCionca** [comment]
Adrian Moldovan
role: Automation Tester with Python + Scraping
discord: Moldovan Adrian
